avro2py/avro_types.py

```diff
@@ -60,12 +61,16 @@ class LogicalType(NamedTuple):
 class Record(NamedTuple):
     """http://avro.apache.org/docs/1.10.0/spec.html#Records"""
     original_schema: Dict
+    resolved_schema: Dict  # after resolving schema reference from original_schema
     name: str
     namespace: str
     fields: List['Field']
     doc: Optional[str] = None
     aliases: List[str] = []
 
+    def fully_qualified_name(self):
+        """Return fully qualified name."""
+        return f'{self.namespace}.{self.name}'
```

Comment:
Rather than a method, let's just make this an `@property`:
```suggestion
    @property
    def fully_qualified_name(self):
        """Return fully qualified name."""
        return f'{self.namespace}.{self.name}'
```
Additionally, if we're going to introduce this, we should be using it in all loci that require the fully-qualified name; https://github.com/hchasestevens/avro2py/blob/master/avro2py/codegen.py#L312 , for example.

Reply:
Sure. But since line #312 is applicable to both `Record` and `Enum`. I have added this property in `Enum` as well.